server/validation/schemaBuildings.js

```diff
@@ -0,0 +1,34 @@
+const yup = require('yup');
+
+const schemaBuildings = yup.object().shape({
+  city: yup
+    .mixed()
+    .oneOf(['Morecambe', 'Hastings'])
+    .required(),
+  latitude: yup.number().required(),
+  longitude: yup.number().required(),
+  address: yup.string().required(),
+  thumbnail: yup.string().required(),
+  owner: yup.string().required(),
+  isOwnerLocal: yup.mixed().oneOf(['Yes', 'No', 'N/A']),
+  previousUse: yup.string().default('N/A'),
+  preferredUse: yup.string(),
+  emptyPeriod: yup.string(),
+  extraInfo: yup.string(),
+  approved: yup
+    .bool()
+    .required()
+    .default(false),
```

Comment:
is it possible to do both `required` and `default`? 
approved isn't actually required as it will always be `false`